### ISSUE TITLE
Multi camera not working and not showing

### ISSUE THREAD

**Issue by user97116** [Apparence-io/CamerAwesome#376]

**Comment by istornz** [comment]
Hello, what device are you using ? Android / iOS ?

Thanks !

**Comment by user97116** [comment]
@istornz Android

**Comment by istornz** [comment]
@user97116 Thanks for your answer 👍 

What is your phone model & Android version ?

Thanks in advance,

**Comment by user97116** [comment]
@istornz Check this in this showing and working multi camera fine here https://github.com/Kurun-pan/android-multi-camera

**Comment by istornz** [comment]
Hello @kyrill-bo Multi camera are not supported on all device... Can you can check what `isMultiCamSupported()` method return ?

Thanks 👍 

Hello @user97116, this repository use Camera2 instead of CameraX which is used on Camerawesome.

**Comment by istornz** [comment]
@kyrill-bo Yeah, BeReal do not display both camera previews... Camerawesome need to display 2 previews at once so we need to use multi cam feature.
